### ISSUE TITLE
Add inter-device networking configuration

### ISSUE THREAD

**Issue by jakeisnt** [jakeisnt/nix-cfg#21]
After all of the devices I'll be using for the near future are properly configured, I should hook them up with a proper inter-device configuration to save me some manual work.

This configuration from [Graham's config](https://github.com/grahamc/nixos-config/blob/master/devices/petunia/configuration.nix) looks useful:
``` nix
ssh = {
      knownHosts.ogden = {
        hostNames = [ "ogden" "10.10.2.15" ];
        publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHICIvUL8AAPDjwP0wUdYADwWSWBieS8iTgNPVa+fynN";
      };
      knownHosts.kif = {
        hostNames = [ "kif.wg.gsc.io" "kif" "10.10.2.16" ];
        publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEjBFLoalf56exb7GptkI151ee+05CwvXzoyBuvzzUbK";
      };

      knownHosts.arm = {
        hostNames = [ "147.75.79.198" ];
        publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDCo+z5d8C6SpCyvC8KAPMAcMEtd5J74tRsk+7sm2KgD";
      };
      extraConfig = ''
        Match User root
          Host 147.75.79.198
          IdentitiesOnly yes
          IdentityFile /rpool/persist/private/root/arm
        Match User root
          Host ogden
          IdentitiesOnly yes
          IdentityFile /rpool/persist/private/root/ogden
        Match User root
          Host kif
          IdentitiesOnly yes
          IdentityFile /rpool/persist/private/root/ogden
        Host rpi1-0
        User root
        HostName 10.5.5.106
        ProxyCommand ssh grahamc@10.5.3.1 nc %h %p
        IdentitiesOnly yes
        IdentityFile /rpool/persist/private/root/rpi
        Host rpi1-1
        User root
        HostName 10.5.5.107
        ProxyCommand ssh grahamc@10.5.3.1 nc %h %p
        IdentitiesOnly yes
        IdentityFile /rpool/persist/private/root/rpi
      '';
    };
  };
```

**Comment by jakeisnt** [comment]
comes after #23 likely

**Comment by jakeisnt** [comment]
closed in favor of #103, which will likely handle this very well
